actions.yaml

```diff
@@ -0,0 +1,4 @@
+show-config:
+  description: Show alertmanager config file.
+show-silences:
```

Comment:
English NitPick: "show-silences" or "show-silenced" ? Same about variable names below.

Reply:
this will show the "silences" not the silenced alarms. AM GUI has a tab named "silences" too